metaserver/src/test/scala/scala/meta/languageserver/ctags/ClasspathCtagsTest.scala

```diff
@@ -0,0 +1,268 @@
+         |_root_.sourcecode.Args.impl. => def impl
+         |_root_.sourcecode.Enclosing# => class Enclosing
+         |_root_.sourcecode.Enclosing. => object Enclosing
+         |_root_.sourcecode.Enclosing.Machine# => class Machine
+         |_root_.sourcecode.Enclosing.Machine. => object Machine
+         |_root_.sourcecode.Enclosing.Machine.impl. => def impl
+         |_root_.sourcecode.Enclosing.impl. => def impl
+         |_root_.sourcecode.File# => class File
+         |_root_.sourcecode.File. => object File
+         |_root_.sourcecode.File.impl. => def impl
+         |_root_.sourcecode.FullName# => class FullName
+         |_root_.sourcecode.FullName. => object FullName
+         |_root_.sourcecode.FullName.Machine# => class Machine
+         |_root_.sourcecode.FullName.Machine. => object Machine
+         |_root_.sourcecode.FullName.Machine.impl. => def impl
+         |_root_.sourcecode.FullName.impl. => def impl
+         |_root_.sourcecode.Impls. => object Impls
+         |_root_.sourcecode.Impls.Chunk# => trait Chunk
+         |_root_.sourcecode.Impls.Chunk. => object Chunk
+         |_root_.sourcecode.Impls.Chunk.Cls# => class Cls
+         |_root_.sourcecode.Impls.Chunk.Def# => class Def
+         |_root_.sourcecode.Impls.Chunk.Lzy# => class Lzy
+         |_root_.sourcecode.Impls.Chunk.Obj# => class Obj
+         |_root_.sourcecode.Impls.Chunk.Pkg# => class Pkg
+         |_root_.sourcecode.Impls.Chunk.Trt# => class Trt
+         |_root_.sourcecode.Impls.Chunk.Val# => class Val
+         |_root_.sourcecode.Impls.Chunk.Var# => class Var
+         |_root_.sourcecode.Impls.enclosing. => def enclosing
+         |_root_.sourcecode.Impls.text. => def text
+         |_root_.sourcecode.Line# => class Line
+         |_root_.sourcecode.Line. => object Line
+         |_root_.sourcecode.Line.impl. => def impl
+         |_root_.sourcecode.Name# => class Name
+         |_root_.sourcecode.Name. => object Name
+         |_root_.sourcecode.Name.Machine# => class Machine
+         |_root_.sourcecode.Name.Machine. => object Machine
+         |_root_.sourcecode.Name.Machine.impl. => def impl
+         |_root_.sourcecode.Name.impl. => def impl
+         |_root_.sourcecode.Pkg# => class Pkg
+         |_root_.sourcecode.Pkg. => object Pkg
+         |_root_.sourcecode.Pkg.impl. => def impl
+         |_root_.sourcecode.SourceCompanion# => class SourceCompanion
+         |_root_.sourcecode.SourceCompanion#apply. => def apply
+         |_root_.sourcecode.SourceCompanion#wrap. => def wrap
+         |_root_.sourcecode.SourceValue# => class SourceValue
+         |_root_.sourcecode.Text# => class Text
+         |_root_.sourcecode.Text. => object Text
+         |_root_.sourcecode.Util. => object Util
+         |_root_.sourcecode.Util.getName. => def getName
+         |_root_.sourcecode.Util.isSynthetic. => def isSynthetic
+         |_root_.sourcecode.Util.isSyntheticName. => def isSyntheticName
+      """.stripMargin
```

Comment:
Now I see what you mean. Yes, we really miss https://facebook.github.io/jest/docs/en/snapshot-testing.html in Scala